src/scheduler/constant.h

```diff
@@ -74,15 +74,15 @@ enum sort_status
 };
 
 
-/* enum used to find out what to skip while searching for the next job to schedule */
+/* enum used to find out what to skip while searching for the next job to schedule.  Values are bits in a bitfield */
 
 enum skip
 {
 	SKIP_NOTHING,
 	/* Value used to know whether reservations are already scheduled or not */
-	SKIP_RESERVATIONS,
+	SKIP_RESERVATIONS = 1,
```

Comment:
did you do this as part of debugging? they should get set to 1 & 2 anyways

Reply:
I added a comment to the enum to explain this.  They are now bits.  While they do get set to 1 and 2, if we need any further bits, it'd be 4.  This is why I set them directly.